pkg/controller/manager/sync_action.go

```diff
@@ -488,3 +489,15 @@ func (a *updateTaskAction) Execute(ctx context.Context, client *mermaidclient.Cl
 
 	return err
 }
+
+// messageOf returns error message embedded in returned error.
+func messageOf(err error) string {
+	err = errors.Cause(err)
+	switch v := err.(type) {
+	case interface {
+		GetPayload() *models.ErrorResponse
+	}:
+		return v.GetPayload().Message
+	}
+	return err.Error()
+}
```

Comment:
why it's moved here? feels like better place is managerclient

Reply:
> feels like better place is managerclient

what do you mean? managerclient is a vendored package

Reply:
right, my bad. I forgot it's vendored now.